packages/core/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe core SDK package will be documented in this file.
 
+## Recent Changes
```

Comment:
Nothing has changed in the "core" package, could you please remove the added lines in this file?